llvm/test/CodeGen/PowerPC/sink-crset-crunset.mir

```diff
@@ -0,0 +1,27 @@
+# RUN: llc -run-pass=machine-sink -mtriple=powerpc64-unknown-linux-gnu -o - %s | FileCheck %s
+
+---
+name:            sink-crset-crunset
+body:             |
+  ; CHECK-LABEL: bb.0.entry
+  bb.0.entry:
+    successors: %bb.1(0x40000000), %bb.2(0x40000000)
+    liveins: $r3
+    ; CHECK-NOT: CRSET
```

Comment:
could you elaborate why MIR test is a better choice over normal codegen test? Is is because CRSET and CRUNSET being pseudo instructions that will be expanded into immediate as you mentioned?

Reply:
I don't think that one can use these instructions in LLVM IR. But maybe I don't understand what a codegen test is?

Reply:
> I don't think that one can use these instructions in LLVM IR

Ah that's a good point. Nvm then.